packages/playground/src/components/filter.vue

```diff
@@ -1,69 +1,82 @@
+          </v-row>
+        </template>
+      </v-expansion-panel-title>
+      <v-expansion-panel-text>
+        <v-form>
           <v-row justify="center" justify-md="start" no-gutters>
-            <v-col
-              cols="12"
-              sm="4"
-              md="2"
-              class="d-flex justify-end align-center ml-2 mr-2 mb-2"
-              v-for="key in Object.keys($props.modelValue)"
-              :key="key"
+            <form-validator
+              v-model="isValidForm"
+              valid-on-init
+              ref="formRef"
+              @update:model-value="$emit('update:valid', $event)"
             >
-              <input-validator
-                v-if="$props.modelValue[key].label"
-                #="{ props }"
-                :rules="$props.modelValue[key].rules?.[0] ?? []"
-                :async-rules="$props.modelValue[key].rules?.[1] ?? []"
-                :value="$props.modelValue[key].value"
+              <v-col
+                cols="12"
+                sm="4"
+                md="2"
+                class="d-flex justify-end align-center ml-2 mr-2 mb-2"
+                v-for="key in Object.keys($props.modelValue)"
+                :key="key"
               >
-                <v-text-field
-                  :disabled="formDisabled"
-                  v-bind="props"
-                  v-model="$props.modelValue[key].value"
-                  :label="$props.modelValue[key].label"
-                  :placeholder="$props.modelValue[key].placeholder"
-                  :type="$props.modelValue[key].type"
-                ></v-text-field>
-              </input-validator>
-            </v-col>
-            <v-col cols="12" sm="4" md="2" class="d-flex justify-start align-center mb-6 ml-4">
+                <input-validator
+                  v-if="$props.modelValue[key].label"
+                  :rules="$props.modelValue[key].value ? $props.modelValue[key].rules?.[0] ?? [] : []"
+                  :async-rules="$props.modelValue[key].rules?.[1] ?? []"
+                  :value="$props.modelValue[key].value"
+                  #="{ props }"
+                  ref="inputRef"
+                >
+                  <v-text-field
+                    v-bind="props"
+                    v-model="$props.modelValue[key].value"
+                    :label="$props.modelValue[key].label"
+                    :placeholder="$props.modelValue[key].placeholder"
+                    :type="$props.modelValue[key].type"
+                    :loading="loading"
+                  ></v-text-field>
+                </input-validator>
+              </v-col>
+            </form-validator>
+          </v-row>
+
+          <v-row>
+            <v-col cols="12" sm="4" md="2" class="d-flex justify-start align-center mb-6 ml-auto mr-4">
               <v-btn
-                :disabled="!isFiltersTouched || formDisabled"
+                class="mr-4"
+                :disabled="!isValidForm"
+                :loading="loading"
```

Comment:
i think we need to set it to disabled not loading
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/62248851/4f6d86f0-2b86-40e6-9e9d-6a64e81cfb19)